app/assets/stylesheets/application.css.scss

```diff
@@ -50,6 +50,21 @@ body {
   background-color: black;
 }
 
+.journal {
+  background-image: url('journal.png');
+  background-size:     cover;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size<br>Colon after property should be followed by one space